### PR TITLE
RadioGroup: isolate Radio component

### PR DESCRIPTION
**Pull request by zignd**
## Context

Isolando componente Radio de dentro do RadioGroup.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 122964083442223f1ab6f8430f2d243a2a4fffea
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 122964083442223f1ab6f8430f2d243a2a4fffea
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 90edcc8493c9ce7281957528c930bc279047b76a
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 90edcc8493c9ce7281957528c930bc279047b76a
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d7b390a96e2f983434eef3bb596de7ccca12af5a
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d7b390a96e2f983434eef3bb596de7ccca12af5a
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8dc69f6da3347aa7318aa245bc782054c46170b8
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8dc69f6da3347aa7318aa245bc782054c46170b8
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: src/RadioGroup/index.js [hunk 1]
```diff
@@ -30,32 +32,19 @@ class RadioGroup extends React.Component {
...
-      <label
-        key={option.value}
-        className={theme.label}
-        htmlFor={`${this.instanceId}-${option.value}-${index}`}
-      >
-        <input
-          type="radio"
-          name={name}
-          value={option.value}
-          id={`${this.instanceId}-${option.value}-${index}`}
```
Comment (vagnervst) [inline src/RadioGroup/index.js:47]: No storybook a seleção dos radios não está funcionando por não termos mais esse id único. Voltando ele já passa a funcionar show, um diff pra exemplificar melhor:

```diff
diff --git a/src/Radio/index.js b/src/Radio/index.js
index 64b7fe1..5f42587 100644
--- a/src/Radio/index.js
+++ b/src/Radio/index.js
@@ -9,6 +9,7 @@ const consumeTheme = ThemeConsumer('UIRadio')
 const Radio = ({
   checked,
   disabled,
+  id,
   label,
   name,
   onChange,
@@ -29,13 +30,13 @@ const Radio = ({
     <label
       key={value}
       className={labelClass}
-      htmlFor={value}
+      htmlFor={id}
     >
       <input
         type="radio"
         name={name}
         value={value}
-        id={value}
+        id={id}
         checked={checked}
         onChange={event => !disabled && onChange(event)}
         disabled={disabled}
@@ -59,6 +60,10 @@ Radio.propTypes = {
    * Disables or enables the radio.
    */
   disabled: PropTypes.bool,
+  /**
+   * Displayed label.
+   */
+  id: PropTypes.string.isRequired,
   /**
    * Displayed label.
    */
diff --git a/src/RadioGroup/index.js b/src/RadioGroup/index.js
index 06d521e..15ed2bd 100644
--- a/src/RadioGroup/index.js
+++ b/src/RadioGroup/index.js
@@ -38,6 +38,7 @@ class RadioGroup extends React.Component {
     const radioButtons = options.map((option, index) => (
       <Radio
         key={`${this.instanceId}-${option.value}`}
+        id={`${this.instanceId}-${option.value}`}
         name={name}
         value={option.value}
         label={option.name}
```

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8dc69f6da3347aa7318aa245bc782054c46170b8
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: f9b7a6973a8e6a66b9a9db11d072c2e084dbbf22
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8dc69f6da3347aa7318aa245bc782054c46170b8
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: f9b7a6973a8e6a66b9a9db11d072c2e084dbbf22
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: src/RadioGroup/index.js (hunk 1 above)
Comment (zignd) [reply src/RadioGroup/index.js:47]: Obrigado pela observação mano! Corrigido! :smile:

**Review by edumoreira1506 — APPROVED** [review]

**Review by edumoreira1506 — APPROVED** [review]

**Review by vagnervst — APPROVED** [review]

**Pull request merged**